src/main/java/com/sender/email/repos/EmailDbProcessing.java

```diff
@@ -0,0 +1,54 @@
+package com.sender.email.repos;
+
+import com.sender.email.models.Email;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.Date;
+import java.util.List;
+
+@Repository
+@Profile("JDBC")
+public class EmailDbProcessing implements EmailProcessing {
+    JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    EmailDbProcessing(DataSource dataSource) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+    }
+
+    @Override
+    public List<Email> getAll() {
+        return jdbcTemplate.query("SELECT * FROM local_db.emails", new EmailMapper());
+    }
+
+    @Override
+    public List<Email> getUnsent() {
+        return jdbcTemplate.query("SELECT * FROM local_db.emails WHERE is_sent = 0", new EmailMapper());
+    }
+
+    @Override
+    public void addNewEmail(List<Email> emails) {
+        jdbcTemplate.update("INSERT INTO local_db.emails (recipient, subject, body, delivery_date, is_sent) " +
```

Comment:
It is strange to pass here _List_ and after take list#get(0).

Pass here single _Email_ entity.